src/tag/tag-filter.component.ts

```diff
@@ -20,7 +21,17 @@ import { Tag } from "./tag.component";
 	`
 })
 export class TagFilter extends Tag {
+	/**
+	 * Function for close/delete the tag
+	 *
+	 */
+	@Output() close: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
We can take advantage of type inference here:
```suggestion
	@Output() close = new EventEmitter<any>();
```